src/main/drivers/timer.c

```diff
@@ -249,7 +251,8 @@ void timerConfigure(const timerHardware_t *timerHardwarePtr, uint16_t period, ui
 {
     configTimeBase(timerHardwarePtr->tim, period, mhz);
     TIM_Cmd(timerHardwarePtr->tim, ENABLE);
-
+if (timerHardwarePtr->tim == TIM7)
+return;
```

Comment:
Sorry, I wrote the code mainly for F103 ... and channel-less timers did not look useful with this driver.
Maybe some necessary properties are not implemented for basic timers, some bugs may surface